File: 2015/src/test/java/nl/sbmf21/aoc15/days/Day13Test.java
package nl.sbmf21.aoc15.days;

import nl.sbmf21.aoc15.AocTest;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Day13Test extends AocTest {

    @Test
    public void testInput() {
        testDay(Day13.class, 618, 601);
    }

    @Test
    public void testExample() {
        assertEquals(330, buildDay(Day13.class, true).part1());
    }
}

File: 2023/kotlin/src/test/kotlin/nl/sbmf21/aoc23/days/Day13Test.kt
package nl.sbmf21.aoc23.days

import nl.sbmf21.aoc.testing.testDay
import org.junit.jupiter.api.Test

class Day13Test {

    @Test
    fun testInput() = testDay(Day13::class.java, 34918, 33054)

    @Test
    fun testExample() = testDay(Day13::class.java, 405, 400, true)
}